core/src/main/java/org/frankframework/management/bus/endpoints/BusEndpointBase.java

```diff
@@ -102,7 +102,7 @@ protected Configuration getConfigurationByName(String configurationName) {
 		}
 		Configuration configuration = getIbisManager().getConfiguration(configurationName);
 		if(configuration == null) {
```

Comment:
Please make changes similar to this in all the `getXxxByName()` methods in this class, not just for configuration.

Reply:
It looks like Yannick also applied this comment.